components/Proposals/Instructions/Decoded/DefaultDecodedInstruction.tsx

```diff
@@ -0,0 +1,53 @@
+import { Text } from '@mantine/core';
+
+import { DecodedInstruction } from '@/lib/instructions/coder/types';
+import { PublicKeyAsCodeLink } from '@/components/Proposals/Instructions/PublicKeyAsCodeLink';
+
+export const DefaultDecodedInstructionCard = ({
+  instruction,
+}: {
+  instruction: DecodedInstruction;
+}) => (
+  <>
+    <Text c="white">
```

Comment:
Nit here, but we do have two themes... You may be able to invoke `const { colorScheme } = useMantineColorScheme();`  `import { useMantineColorScheme } from '@mantine/core';` to help.
<img width="325" alt="Screenshot 2024-03-17 at 5 47 35 AM" src="https://github.com/metaDAOproject/meta-dao-frontend/assets/1007667/3b67d44e-4ca4-4a72-95fa-0251e750b4cf">

Reply:
+1 on this. Looks like you can't see it in light theme at all?

Reply:
ah, good catch - didn't realize we had the themes. straight up excluding the color attribute works fine, it's just a smidge darker on dark theme.

<img width="280" alt="Screenshot 2024-03-17 at 10 50 49" src="https://github.com/metaDAOproject/meta-dao-frontend/assets/10272973/d1471a16-9d32-4fe3-ba08-6b124cab31c8">

<img width="280" alt="Screenshot 2024-03-17 at 10 50 56" src="https://github.com/metaDAOproject/meta-dao-frontend/assets/10272973/4ef4c739-1ad5-4032-8f98-06ae406336e2">